apps/condo/domains/property/components/panels/MapConstructor.spec.ts

```diff
@@ -0,0 +1,145 @@
+import {  
+    mapAddSection,
+    mapUpdateSection,
+    mapRemoveSection,
+
+    mapAddUnit,
+    mapUpdateUnit,
+    mapRemoveUnit,
+
+    getUnitInfo,
+    getPossibleFloors,
+
+    getSectionFloorOptions,
+    getSectionOptions,
+} from './MapConstructor'
+
+
+describe('Map constructor', () => {
```

Comment:
I can't understand the map structure and new logic after the looking inside the spec